src/traversals/bfs.jl

```diff
@@ -302,3 +302,40 @@ Return a vector filled with the geodesic distances of vertices in  `g` from
 For vertices in disconnected components the default distance is -1.
 """
 gdistances(g::AbstractGraph, source) = gdistances!(g, source, Vector{Int}(nv(g)))
+
+"""
+    has_path(g::AbstractGraph, u, v; exclude_vertices=Vector())
+
+Return `true` if there is a path from `u to `v`` in `g` (while avoiding vertices in
+`exclude_vertices``) or `u == v`. Return false if there is no such path or if `u` or `v` is "excluded". 
```

Comment:
minor: if you're editing this anyway, no quotes around `excluded`. Instead, perhaps,
```
Return false if no path exists, or if `u` or `v` is in `excluded_vertices`.
```